app/queries/my_facilities/blood_pressure_control_query.rb

```diff
@@ -61,6 +61,7 @@ def overall_controlled_bps
     @overall_controlled_bps ||=
       LatestBloodPressuresPerPatient
       .where(registration_facility: facilities)
+      .where(patient: overall_patients)
```

Comment:
Neat. We can remove the `.where(registration_facility: facilities)` now too, right?